eng/package.ps1

```diff
@@ -14,7 +14,7 @@ $slnMac = Join-Path $PSScriptRoot ../Microsoft.Maui.Packages-mac.slnf
 
 # Bootstrap ./bin/dotnet/
 $csproj = Join-Path $PSScriptRoot ../src/DotNet/DotNet.csproj
-& dotnet build $csproj -bl:$logsDirectory/dotnet-$configuration.binlog
+#& dotnet build $csproj -bl:$logsDirectory/dotnet-$configuration.binlog
```

Comment:
I think this broke me from building on my dev machine.

Reply:
what s the error you are getting?

Reply:
Mine too:
```
Get-Item: C:\src\maui\eng\package.ps1:21
Line |
  21 |  $dotnet = (Get-Item $dotnet).FullName
     |             ~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\src\maui\bin\dotnet\' because it does not exist.
```

I would be concerned we're not actually building MAUI with the right bits if this is commented out?

If we're not using `./bin/dotnet/dotnet`, for example, what version of the Android workload is being used?

Reply:
we added a previous step that calls to install the workloads, so we can add a retry when installing workloads fails. seems to be working fine on CI, and what i use locally,

Reply:
Note: if you `git clean -xdf`, the `bin` directory is deleted. Then doing `dotnet tool restore` and `dotnet cake --configuration="Release"` produces the error @jonathanpeppers has above.

Reply:
https://github.com/dotnet/maui/pull/4775/files#diff-b295f649a0c373e6e5bc469fec1fddbd34b6b907a21927be10241829610cb604R19

This installs the workloads first and then calls pack, just as other stages to (for full build, or for build the templates)

Reply:
So are we not supposed to use `dotnet cake` to build the repo anymore?

Reply:
Well doing locally should be something like: 

`dotnet cake --target=VS-NET6`

Reply:
what's the error you are getting?

Reply:
```
Get-Item: C:\src\maui\eng\package.ps1:21
Line |
  21 |  $dotnet = (Get-Item $dotnet).FullName
     |             ~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\src\maui\bin\dotnet\' because it does not exist.
```